apps/passport-client/src/dispatch.ts

```diff
@@ -129,6 +131,11 @@ export type Action =
       subscriptionId: string;
       onSucess?: () => void;
       onError?: (e: Error) => void;
+    }
+  | {
+      type: "merge-import";
+      collection: PCDCollection;
+      pcdsToMergeIds: Set<PCD["id"]>;
```

Comment:
This reminds me of something i've been meaning to add an issue for: https://github.com/proofcarryingdata/zupass/issues/1354